### PR TITLE
Add resources strip to /openstack

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add resources strip to /openstack

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
- Check that the new 'resources' section has been added and is as per copy doc

## Issue / Card

Fixes: #2407 
Doc: https://docs.google.com/document/d/12z6QEGpqMVbNPJOd5aUfn0VnXQ0Sa5YriEBLA7VGNwI/edit#

### CODE REVIEW

**Comment by frankiegranato** [comment]
I've reduced the word count in each headline. Please update from Copydoc. Sorry!

**Comment by grahambancroft** [comment]
@frankiegranato Done http://www.ubuntu.com-pr-2418.run.demo.haus/openstack

**Comment by grahambancroft** [comment]
@frankiegranato I've changed it on the Foundation Cloud also https://github.com/canonical-websites/www.ubuntu.com/pull/2417

**Comment by grahambancroft** [comment]
@frankiegranato approves

File: redirects.yaml [hunk 1]
```diff
@@ -1608,6 +1606,7 @@ ubuntu-advantage-storage/?: "/cloud/storage"
 ubuntu-core/?: "/core"
 ubuntu-releases/14/?: "/download/desktop"
 ubuntu-releases/14\.04/ubuntu-14\.0/?: "/download/desktop"
+ubuntu-releases/14/?: "/download/desktop"
```
Comment (pmahnke) [inline redirects.yaml]: isn't this the same as line 1607?

File: redirects.yaml [hunk 2]
```diff
@@ -734,8 +734,6 @@ files/u3/Contact\.pdf/?: "https://insights.ubuntu.com/"
 files/wikimedia_casestudy_v1\.pdf/?: "https://insights.ubuntu.com/"
 forum/?: "https://community.ubuntu.com/"
 forums/?: "https://community.ubuntu.com/"
-foundation-cloud/?: "/cloud/foundation-cloud"
```
Comment (pmahnke) [inline redirects.yaml]: don't we want these?

**Comment by grahambancroft** [comment]
@pmahnke I've fixed the redirects

**Pull request merged**